addons/event_sale/models/event_event.py

```diff
@@ -11,14 +11,14 @@ class Event(models.Model):
         'sale.order.line', 'event_id',
         groups='sales_team.group_sale_salesman',
         string='All sale order lines pointing to this event')
-    sale_price_subtotal = fields.Monetary(
-        string='Sales (Tax Excluded)', compute='_compute_sale_price_subtotal',
+    sale_price_total = fields.Monetary(
+        string='Sales (Tax Included)', compute='_compute_sale_price_total',
         groups='sales_team.group_sale_salesman')
 
     @api.depends('company_id.currency_id',
                  'sale_order_lines_ids.price_subtotal', 'sale_order_lines_ids.currency_id',
```

Comment:
shouldn't we use `sale_order_lines_ids.price_total` here instead to be coherent with the changes ?